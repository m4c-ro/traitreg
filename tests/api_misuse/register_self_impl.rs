fn main() {}



struct MyStruct;

#[traitreg::register]
impl MyStruct {}
