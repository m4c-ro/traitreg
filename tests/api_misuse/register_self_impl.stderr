error: custom attribute panicked
 --> tests/api_misuse/register_self_impl.rs:7:1
  |
7 | #[traitreg::register]
  | ^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Can only register an implementation of a trait, 'impl <Trait> for <Type>'.
